_tools/check_docs.ts

```diff
@@ -84,7 +85,7 @@ function assertHasReturnTag(document: { jsDoc: JsDoc; location: Location }) {
   const tag = document.jsDoc.tags?.find((tag) => tag.kind === "return");
   if (tag === undefined) {
     diagnostics.push(
-      new DocumentError("Symbol must have a @return tag", document),
+      new DocumentError("Symbol must have a @returns tag", document),
```

Comment:
Nit: this is the most true.
```suggestion
      new DocumentError("Symbol must have a @return or @returns tag", document),
```

Reply:
Maybe we should standardize on just `@returns`, which is mostly used?